src/file/model/baseFile.ts

```diff
@@ -0,0 +1,12 @@
+import * as mongoose from 'mongoose';
+import { Document } from 'mongoose';
+import { baseModelName } from "./config"
+
+interface IBaseFile {
+    fileModel: string;
```

Comment:
Change it also when you change the `discriminatorKey` in the config.ts

Reply:
done